sentry-rails/lib/sentry/rails/active_job.rb

```diff
@@ -79,6 +79,12 @@ def sentry_context(job)
 
           def sentry_serialize_arguments(argument)
             case argument
+            when Range
+              if (argument.begin || argument.end).is_a?(ActiveSupport::TimeWithZone)
+                argument.to_s
+              else
+                argument.map { |v| sentry_serialize_arguments(v) }
```

Comment:
Do we need to serialize individual components in the range? If the range is `(1..10000)` then it'd probably be pretty slow AND it'd allocate a new array with 10k elements in it.
Maybe in this case we just leave the `argument` until we find another element type that requires special treatments?

Reply:
@st0012 we don't but it's the original behavior - is it OK to change it in a patch/bugfix release?

Reply:
@st0012 yes exactly :)